### PR TITLE
Chore: docs grammar fix

### PR DESCRIPTION
**Pull request by tusharxoxoxo**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Backstage uses Snyk vulnerability scans in order to make sure we minimize vulner
 
 There are many situations where a vulnerability does not affect a particular dependency because of how the vulnerable package is used. In that situation the package authors may choose to stay at the current version rather than bumping the dependency, leading to a warning in the vulnerability scans but no actual vulnerability.
 
-To work around this and other similar issues, Snyk provides a method to ignore vulnerabilities. In order to provide the best visibility and most utility to adopters of Backstage, we store these ignore rules in `.snyk` policy files. This allows adopters to rely on our ignore policies if they wish to do so.
+To work around this and other similar issues, Snyk provides a method to ignored vulnerabilities. In order to provide the best visibility and most utility to adopters of Backstage, we store these ignore rules in `.snyk` policy files. This allows adopters to rely on our ignore policies if they wish to do so.
```
Comment (jhaals) [inline SECURITY.md:34]: This should remain `ignore` :)

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for improving the documentation! 👏 Small tweak and this is good to go

File: SECURITY.md (hunk 1 above)
Comment (tusharxoxoxo) [reply SECURITY.md:34]: ok

**Comment by tusharxoxoxo** [comment]
my bad

**Comment by tusharxoxoxo** [comment]
is it right this time??

**Comment by tusharxoxoxo** [comment]
Screenshot 2022-09-21 at 11 22 23
I checked the git log 
it's showing that it's signed off now

**Review by jhaals — CHANGES_REQUESTED** [review]
hmm it looks like the DCO signing is not quite right, looks alright from your screenshot, there's probably some commit further back in the chain that is unsigned. The DCO bot suggests that you `git rebase HEAD~6 --signoff` and you have to force push afterwards. Looks like the branch need a rebase too.

If you don't manage to get the CI status green it's probably easier to close this PR and start a new branch from a fresh master.

Good luck! 🙏

**Comment by tusharxoxoxo** [comment]
2 CI /test and DCO failed

**Review by jhaals — CHANGES_REQUESTED** [review]
I can still see that some of the earlier commits are not DCO signed and the CI error should be fixed by rebasing. I suggest you open a new PR with newly updated master from the upstream if you are unsure how to update the existing branch

**Comment by tusharxoxoxo** [comment]
Ok

**Pull request closed without merging**